portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +36,67 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> quotes;
+
+  @Override
+  public void init() {
+  }
+
+
+  // Generates a randome quote from the ArrayList Quotes and converts the quote to a JSON object.
```

Comment:
Prefer javadoc quotes for method descriptions

/**
 * Generates a randome quote from the ArrayList Quotes and converts the quote to a JSON object.
 */

Also `randome` --> `random`

Example here: https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html